src/vibe/components/SidebarNav/components/NavSingleItem.js

```diff
@@ -18,9 +18,16 @@ const NavSingleItem = ({ item }) => {
       </li>
     );
   } else {
+    let url = item.url;
+
+    // Force relative URLs to start with a slash
+    if (url.substring(0, 1) !== '/') {
```

Comment:
Would `url.charAt(0) !== '/'` be a little bit more readable? 

also a one-liner would be more concise but I don't mind the if statement either. 

``const url = item.url.charAt(0) !== '/' ? `/${item.url}` : item.url; ``

What do you think @cstuder?

Reply:
Agreed.

I inverted the expression to make it a little bit more readable, in my opinion.